### ISSUE TITLE
virtual keyboard lag

### ISSUE THREAD

**Issue by danboid** [ccherrett/oom#59]
When used in conjunction with the Noisemaker LV2 plugin at least, the on-screen keyboard of the performer window isn't very responsive in that it seems to only manage about (just over) 1 click per second.

**Comment by ccherrett** [comment]
What is the full spec of your machine? It is running fine on 4 machines tested and I am wondering if it is a low spec machine.

Thanks!

**Comment by danboid** [comment]
Hi Chris!

The full(ish) spec of the machine I reported this on is a 3Ghz i3 with 4GB RAM a 1TB SATA HD running Deb Testing amd64 but I've just tried this more thoroughly again but on a 1.9Ghz Turion64x2 with 2GB RAM and I found that not only could I click along to a 140/145 bpm metronome without any notes getting dropped but that it virtual keyboard performed much better than a certain other Linux DAW.

Its only when I get to about 150bpm that Noisemaker begins to lag on me so I think so this is acceptable performance for clicking on a virtual keyboard with an average mouse so I think we're safe to close this now! :)
